src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -321,14 +325,18 @@ private String putMainImageReferenceInBodyNode(Node bodyNode, String mainImageUU
         return getNodeAsHTML5String(bodyNode);
     }
 
-    private Design extractDesign(final XPath xPath, final Document eomFileDoc) throws XPathExpressionException {
-        final String theme = Strings.nullToEmpty(xPath.evaluate("/doc/lead/lead-components/content-package/@design-theme", eomFileDoc)).trim();
-
-        if (Strings.isNullOrEmpty(theme)) {
+    private Design extractDesign(final XPath xPath, final Document valueDoc, final Document attributesDoc) throws XPathExpressionException {
+        final String designThemeOld = Strings.nullToEmpty(xPath.evaluate(XPATH_DESIGN_THEME_OLD, valueDoc)).trim().toLowerCase();
+        final String designThemeNew = Strings.nullToEmpty(xPath.evaluate(XPATH_DESIGN_THEME, attributesDoc)).trim().toLowerCase();
```

Comment:
I would just call this variable `designTheme` and remove the following line.

Reply:
shortened.